pkg/microservice/policy/core/repository/mongodb/healthz.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2021 The KodeRover Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mongodb
+
+import (
+	"context"
+
+	mongotool "github.com/koderover/zadig/pkg/tool/mongo"
+)
+
+// CheckMongodbHealth
+func CheckMongodbHealth(ctx context.Context) error {
```

Comment:
it is not part of repo layer, move to pkg/microservice/policy/core/service/healthz.go